index.js

```diff
@@ -471,7 +471,7 @@ const plugin = fp(async function (app, opts) {
       document = cached.document
     }
 
-    if (reply && reply.request.raw.method === 'GET') {
+    if (reply && reply.request.raw.method === 'GET' && reply.request.headers.upgrade?.toLowerCase() !== 'websocket') {
```

Comment:
```suggestion
    if (reply && reply.request.raw.method === 'GET' && !reply.request.ws) {
```

Reply:
This should work, check it out!

Reply:
Thanks, it worked with manual testing, so I updated the corresponding test to set that flag on the request like `fastify-websocket` does.